### PR TITLE
adding builds for go1.12/13/14

### PR DESCRIPTION
**Pull request by luthermonson**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,6 +1,64 @@
...
+name: windows-go114-1809
+
+platform:
+  os: windows
+  arch: amd64
+  version: 1809
+
+steps:
+- name: docker-ci
+  image: plugins/docker
```
Comment (deniseschannon) [inline .drone.yml:12]: Was this changed into a manifest by them to support the different arch?

File: .drone.yml [hunk 2]
```diff
@@ -58,7 +116,301 @@ volumes:
...
+name: windows-go114-1909
+
+platform:
+  os: windows
+  arch: amd64
+  version: 1909
+
+steps:
+- name: docker-ci
+  image: plugins/docker
```
Comment (deniseschannon) [inline .drone.yml:128]: Does this need Windows support and does it have 1909 support?

File: .drone.yml [hunk 3]
```diff
@@ -58,7 +116,301 @@ volumes:
...
+name: windows-go113-1909
+
+platform:
+  os: windows
+  arch: amd64
+  version: 1909
+
+steps:
+- name: docker-ci
+  image: plugins/docker:windows-1909-amd64
```
Comment (deniseschannon) [inline .drone.yml]: why does this one use a specific tag but not the go 1.14 one?

File: .drone.yml [hunk 4]
```diff
@@ -58,7 +116,301 @@ volumes:
...
+      from_secret: docker_username
+  volumes:
+  - name: docker_pipe
+    path: \\\\.\\pipe\\docker_engine
+  when:
+    event:
+    - pull_request
+
+- name: docker-publish
+  image: plugins/docker:windows-1909-amd64
```
Comment (deniseschannon) [inline .drone.yml]: why does this one use a specific tag but not the go 1.14 one?

File: .drone.yml [hunk 5]
```diff
@@ -58,7 +116,301 @@ volumes:
...
+kind: pipeline
+name: manifests-go113
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: push-manifest
+  image: plugins/manifest:1.1.0
```
Comment (deniseschannon) [inline .drone.yml]: should we update to darren's image?

File: .drone.yml [hunk 6]
```diff
@@ -58,7 +116,301 @@ volumes:
...
+kind: pipeline
+name: manifests-go114
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: push-manifest
+  image: plugins/manifest:1.1.0
```
Comment (deniseschannon) [inline .drone.yml]: should we update to darren's image?

File: .drone.yml [hunk 7]
```diff
@@ -58,7 +116,301 @@ volumes:
...
+  settings:
+    ignore_missing: true
```
Comment (deniseschannon) [inline .drone.yml]: remove this line

File: .drone.yml [hunk 8]
```diff
@@ -58,7 +116,301 @@ volumes:
...
```
Comment (deniseschannon) [inline .drone.yml]: remove this line

File: .drone.yml [hunk 9]
```diff
@@ -70,13 +422,13 @@ steps:
   image: plugins/docker:windows-1809-amd64
```
Comment (deniseschannon) [inline .drone.yml]: why does this one use a specific tag but not the go 1.14 one?

File: .drone.yml [hunk 10]
```diff
@@ -89,13 +441,129 @@ steps:
 - name: docker-publish
   image: plugins/docker:windows-1809-amd64
```
Comment (deniseschannon) [inline .drone.yml]: why does this one use a specific tag but not the go 1.14 one?

File: .drone.yml [hunk 11]
```diff
@@ -89,13 +441,129 @@ steps:
...
+name: windows-go112-1903
+
+platform:
+  os: windows
+  arch: amd64
+  version: 1903
+
+steps:
+- name: docker-ci
+  image: plugins/docker:windows-1903-amd64
```
Comment (deniseschannon) [inline .drone.yml]: why does this one use a specific tag but not the go 1.14 one?

File: .drone.yml [hunk 12]
```diff
@@ -89,13 +441,129 @@ steps:
...
+      from_secret: docker_username
+  volumes:
+  - name: docker_pipe
+    path: \\\\.\\pipe\\docker_engine
+  when:
+    event:
+    - pull_request
+
+- name: docker-publish
+  image: plugins/docker:windows-1903-amd64
```
Comment (deniseschannon) [inline .drone.yml]: why does this one use a specific tag but not the go 1.14 one?

File: .drone.yml [hunk 13]
```diff
@@ -89,13 +441,129 @@ steps:
...
+name: windows-go112-1909
+
+platform:
+  os: windows
+  arch: amd64
+  version: 1909
+
+steps:
+- name: docker-ci
+  image: plugins/docker:windows-1909-amd64
```
Comment (deniseschannon) [inline .drone.yml]: why does this one use a specific tag but not the go 1.14 one?

File: .drone.yml [hunk 14]
```diff
@@ -89,13 +441,129 @@ steps:
...
+      from_secret: docker_username
+  volumes:
+  - name: docker_pipe
+    path: \\\\.\\pipe\\docker_engine
+  when:
+    event:
+    - pull_request
+
+- name: docker-publish
+  image: plugins/docker:windows-1909-amd64
```
Comment (deniseschannon) [inline .drone.yml]: why does this one use a specific tag but not the go 1.14 one?

File: .drone.yml [hunk 15]
```diff
@@ -129,7 +597,7 @@ steps:
     ignore_missing: true
```
Comment (deniseschannon) [inline .drone.yml]: remove this line

**Review by deniseschannon — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 2 above)
Comment (luthermonson) [reply .drone.yml:128]: yes i added it

File: .drone.yml (hunk 14 above)
Comment (luthermonson) [reply .drone.yml]: mistake, fixed

File: .drone.yml (hunk 13 above)
Comment (luthermonson) [reply .drone.yml]: mistake, fixed

File: .drone.yml (hunk 15 above)
Comment (luthermonson) [reply .drone.yml]: removed everywhere, switched to latest too as darrens changes are merged in

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**